docs/best_practices/tool_xml.rst

```diff
@@ -164,6 +164,24 @@ The help tag should be started and finished by a CDATA tag.
 
 `http://en.wikipedia.org/wiki/CDATA <http://en.wikipedia.org/wiki/CDATA>`__
 
+Inside the help tag you should describe the functionality of your tool.
+The help tag is to the ``help=""`` attribute as a man page is to the ``--help``
+flag. The help tag should cover the tools functionality, use cases, and even
```

Comment:
``The help tag... flag``: this comparison looks a bit contrived to me, I'd remove it.